auto_ml/utils_scoring.py

```diff
@@ -35,6 +37,14 @@ def advanced_scoring_classifiers(probas, actuals, name=None):
             predicted_labels.append(0)
     print(format(accuracy_score(y_true=actuals, y_pred=predicted_labels) * 100, '.1f') + '%')
 
+    
+    print('\nHere is a confusion matrix showing predictions and actuals by label')
+    #it would make sense to use sklearn's confusion_matrix here but it apparently has no labels
```

Comment:
I figured I should explain why I wasn't using the built-in function. Conveniently, the guy on StackExchange basically solved it.